### PR TITLE
Covering connect method with span

### PR DESCRIPTION
**Pull request by rajeshkarnena**
Covering connect method with span for better insights on how long acquiring a connection takes. 

This PR is in reference to [comment](https://github.com/XSAM/otelsql/issues/36#issuecomment-1068459275) on #36

### CODE REVIEW

File: methods.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ type Method string
 type Event string
 
 const (
+	MethodConnConnect      Method = "sql.conn.connect"
```
Comment (XSAM) [inline methods.go:24]:
```suggestion
	MethodConnectorConnect Method = "sql.connector.connect"
```

This method belongs to `otConnector`, so `sql.connector.connect` could be a better name.

**Review by XSAM — COMMENTED** [review]
Please update `CHANGELOG.md` to reflect this feature.

File: methods.go (hunk 1 above)
Comment (rajeshkarnena) [reply methods.go]: My bad, renamed it.

**Comment by rajeshkarnena** [comment]
@XSAM Thanks for the review, fixed the name and updated the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Changed
+
+- Covering connector's connect method with span. (#66)
+
```
Comment (XSAM) [inline CHANGELOG.md:14]:
```suggestion
### Added

- Covering connector's connect method with span. (#66)

```

**Review by XSAM — COMMENTED** [review]
Looks good overall. Only one nit to fix.

**Review by XSAM — APPROVED** [review]
@rajeshkarnena Thanks for your contribution.

**Comment by rajeshkarnena** [comment]
@XSAM Thanks for accepting and merging this. May I know when will this be released?

**Comment by XSAM** [comment]
I will release this tomorrow along with the new OTel version.

**Pull request merged**